vue-frontend/src/components/jobs/CareerView.vue

```diff
@@ -0,0 +1,291 @@
+          </collapse-transition>
+          <div class="career-divider" v-if="career.id != careers.length"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import CollapseTransition from "@ivanv/vue-collapse-transition/src/CollapseTransition.vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import axios from "axios";
+import config from "@/config.js";
+
+import { faApple, faAndroid } from "@fortawesome/free-brands-svg-icons";
+import {
+  faGlobe,
+  faBullhorn,
+  faPenNib,
+} from "@fortawesome/free-solid-svg-icons";
+
+export default {
+  data() {
+    return {
+      careers: [],
+      currentCareerIndex: 0,
+      isOpenCareerList: true,
+      previousCareerIndex: null,
+    };
+  },
+  components: {
+    CollapseTransition,
+    FontAwesomeIcon,
+  },
+  mounted() {
+    this.getCareer();
+  },
+  methods: {
+    openCareer(index) {
+      if (this.previousCareerIndex == index && this.isOpenCareerList) {
+        this.isOpenCareerList = false;
+      } else {
+        this.isOpenCareerList = true;
+      }
+
+      this.currentCareerIndex = index;
+      this.previousCareerIndex = this.currentCareerIndex;
+    },
+    getCareer() {
+      axios
+        .get(config.API_BASE + "/api/careers")
+        .then((response) => {
+          this.careers = response.data;
+          this.careers[0].icon_name = faApple;
+          this.careers[1].icon_name = faAndroid;
+          this.careers[2].icon_name = faGlobe;
+          this.careers[3].icon_name = faBullhorn;
+          this.careers[4].icon_name = faPenNib;
+          this.careers[5].icon_name = faApple;
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.container {
+  margin: 100px auto;
+}
+
+.description {
+  margin: 30px 10px;
+  color: #3d3d3d;
+}
+
+.text-weight {
+  font-weight: 700;
+}
+
+.career-container {
+  border-radius: 0.9rem;
+  border: 1px solid #e2e2e2;
+  overflow: hidden;
+  margin: 30px 10px;
+}
+
+.career-list-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.career-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.career-divider {
+  background: #e2e2e2;
+  height: 1px;
+}
+
+.career-header {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+  padding: 30px;
+  cursor: pointer;
+}
+
+.career-icon {
+  width: 2.3rem;
+}
+
+.career-summary {
+  padding: 30px;
+}
+
+.career-title {
+  font-style: normal;
+  font-weight: 700;
+  color: #3d3d3d;
+}
+
+.career-plus-icon {
+  margin-left: auto;
+  order: 2;
+  color: rgba(61, 61, 61, 0.5);
+}
+
+.read-apply-btns {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0 0 30px;
+}
+
+.gradient-btn {
+  margin: 5px;
+}
+
+.gradient-btn > span {
+  font-size: 1.125rem;
+  line-height: 1.364rem;
+}
+
+.gradient-career-icon {
+  color: #f2709c;
+}
+
+.gradient-border-btn {
```

Comment:
I think we reused gradient-border-btn in the whole website. Why do we need to take it separately here? We can override the CSS of that btn if needed